File: docs/db/entities/person/Person.cs
namespace Nicole.Database.Entities;

public sealed class Person
{
    public Guid PersonId { get; set; }
    public string IdentificationType { get; set; } = string.Empty;
    public string Identification { get; set; } = string.Empty;
    public string PersonType { get; set; } = string.Empty;
    public string? LastName { get; set; }
    public string? MiddleName { get; set; }
    public string? FirstName { get; set; }
    public string? BusinessName { get; set; }
    public string? Gender { get; set; }
    public DateOnly? BirthDate { get; set; }
    public string? Address { get; set; }
    public string? Phone { get; set; }
    public string? Email { get; set; }
    public string? Province { get; set; }
    public int? City { get; set; }
    public string? Remarks { get; set; }
    public string Status { get; set; } = "A";
    public string CreatedBy { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public string? UpdatedBy { get; set; }
    public DateTime? UpdatedAt { get; set; }
}

File: docs/db/entities/person/person.ts
export interface Person {
  personId: string;
  identificationType: string;
  identification: string;
  personType: string;
  lastName: string | null;
  middleName: string | null;
  firstName: string | null;
  businessName: string | null;
  gender: string | null;
  birthDate: string | null;
  address: string | null;
  phone: string | null;
  email: string | null;
  province: string | null;
  city: number | null;
  remarks: string | null;
  status: string;
  createdBy: string;
  createdAt: string;
  updatedBy: string | null;
  updatedAt: string | null;
}
